lars-reimann/api-editor
# fix: missing entries in selection view

Closes #270.

### Summary of Changes

Show entries in the selection view even when they don't have children. When filtering, entries only get removed when we filter at a lower API level, i.e. we **keep** all
* modules that match the filter if we only filter modules (e.g., `module:_min_` finds the empty module `sklearn._min_dependencies`).
* classes that match the filter if we only filter modules or classes (e.g., `class:ArffCon` finds the empty class `ArffContainerType`).
* functions that match the filter if we only filter modules, classes, or parameters (e.g., `function:get_con` finds the parameterless function `get_config`.

### Testing instructions

1. Check that the selection view contains entries without children when no filter is set.
2. Try the examples given above and others of the same category.
3. Check that non-matching entries are still removed properly when filtering.
